### DISCUSSION TITLE
HP 24w not support on Windows 11 21H2

### DISCUSSION THREAD

**Discussion by ghost** [xanderfrangos/twinkle-tray#212] [Monitor Compatibility Issues]
HP 24w not support on Windows 11 21H2, just not support in twinkle-tray but supported in other DDC controlled program like https://github.com/nik9play/tinyBrightness

**Comment by ghost** [comment]
Run twinkle-tray as administrator, HP 24w detected and works fine.

**Reply by siddhantbapna** [reply]
Hey, have you encountered this issue again any time after your above reply?

**Comment by flyingfeather99** [comment]
HP 24w and Win10 21h1
Works perfectly if i install twinkle-tray from github release, but not from Microsoft store.
